.travis/metaSMT.sh

```diff
@@ -6,6 +6,7 @@ sudo apt-get -y install libboost1.55-dev libz3 libz3-dev libgmp-dev
 # Clone
 git clone git://github.com/hoangmle/metaSMT.git
 cd metaSMT
+git checkout 1f9e399
 git clone git://github.com/agra-uni-bremen/dependencies.git
```

Comment:
I consider they are stable enough, but a fixed commit is being used now so we are on the safe side.